airsonic-main/src/main/java/org/airsonic/player/service/MediaFileService.java

```diff
@@ -412,11 +412,22 @@ private void updateChildren(MediaFile parent) {
         mediaFileDao.createOrUpdateMediaFile(parent);
     }
 
+    public boolean includeMediaFile(MediaFile candidate) {
+        return includeMediaFile(candidate.getFile());
+    }
+
+    public boolean includeMediaFile(File candidate) {
+        String suffix = FilenameUtils.getExtension(candidate.getName()).toLowerCase();
+        if (!isExcluded(candidate) && (FileUtil.isDirectory(candidate) || isAudioFile(suffix) || isVideoFile(suffix))) {
```

Comment:
Nit: Splitting the condition into two if improves a bit the readability. Feel free to ignore if you disagree :)

```suggestion
        if (!isExcluded(candidate) {
           if (FileUtil.isDirectory(candidate) || isAudioFile(suffix) || isVideoFile(suffix)) {
```

Reply:
I just moved those two lines from the other function without changing them. I agree splitting would improve readability, I just usually tend to separate functional vs cleanup changes (to minimize changes and simplify reviewing).